test/e2e/apimachinery/resource_quota.go

```diff
@@ -528,6 +530,11 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		err = waitForResourceQuota(ctx, f.ClientSet, f.Namespace.Name, quotaName, usedResources)
 		framework.ExpectNoError(err)
 
+		ginkgo.By("Not allowing a PersistentVolumeClaim to be created that exceeds remaining quota")
```

Comment:
is this test change related to the InPlacePodVerticalScaling change? I don't see how it could be, since it's doing a create operation and the bugfix is around update admission

Reply:
If this is an unrelated test change, split this out into a separate PR